docker-compose.yml

```diff
@@ -9,6 +9,8 @@ services:
       FIO_API_TOKEN: /fio-api-token
       # if FIO_API_TOKEN is provided, assign devices to a device-group
       DEVICE_GROUP: ${DEVICE_GROUP-}
+      DEVICE_REGISTRATION_API: ${DEVICE_REGISTRATION_API-https://default-api-domain.com/ota/devices/}
```

Comment:
Let's not give an invalid default. 90% of the people using this will want api.foundries.io.